vpr/src/place/initial_placement.cpp

```diff
@@ -592,6 +802,7 @@ static vtr::vector<ClusterBlockId, t_block_score> assign_block_scores() {
 
     //go through all blocks and store floorplan constraints score
```

Comment:
Can update this comment (not just floorplan constraints) or move your initialization into its own loop.